napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -333,22 +528,40 @@ def addItem(
                 self.handle_action, item, pkg_name, InstallerActions.CANCEL
             )
         )
+
         item.setSizeHint(widg.sizeHint())
-        self.setItemWidget(item, widg)
+        widg.install_info_button.setDuration(0)
+        widg.install_info_button.toggled.connect(
+            lambda: self._resize_pluginlistitem(item)
+        )
+
+    def _resize_pluginlistitem(self, item):
+        """Resize the plugin list item, especially after toggling QCollapsible."""
+        height = item.widget.height()
+        if item.widget.install_info_button.isExpanded():
+            item.widget.setFixedHeight(int(height * 1.6))
```

Comment:
Why 1.6? It is tested on all systems and with an enabled scaling interface (ex on windows)?

Reply:
I checked this on windows (with and without UI scaling), ubuntu and mac and it seems to scale just fine, so I'm not too concerned about this magic number, though maybe we bring it out into a constant. Do you have a better suggestion for how to scale this widget?